tests/dbt/test_graph.py

```diff
@@ -312,3 +313,40 @@ def test_load_via_load_via_custom_parser(pipeline_name):
     assert dbt_graph.nodes == dbt_graph.filtered_nodes
     # the custom parser does not add dbt test nodes
     assert len(dbt_graph.nodes) == 8
+
+
+@pytest.mark.integration
+@pytest.mark.parametrize("load_method", ["load_via_dbt_ls", "load_from_dbt_manifest"])
+def test_load_dbt_ls_and_manifest_with_model_version(load_method):
```

Comment:
@binhnq94, what do you think we are having some tests to confirm  `dbt run with --models <name>` runs as expected?
ATM, we don't have a test matrix for `dbt`, but this type of change makes me wonder if we should.

Reply:
Do you give me sample kind of tests ? I will add it as the same.

Reply:
@ binhnq94 BTW: I suggest we do this in a separate PR. You've already done great with the alias support here!

I thought about something along the lines of what we did here, but without extending the `test` environment, but using `test_dbt` or something similar:
https://github.com/astronomer/astronomer-cosmos/blob/f2dd7e129423a5a4ed0000835951809a74e08577/pyproject.toml#L142-L153

We could decorate a subset of relevant tests with this "dbt" marker, running only the marked dbt tests using something similar to what we did for the integration tests, validating behaviours that will change depending on the version of dbt (such as node name selection):
https://github.com/astronomer/astronomer-cosmos/blob/main/pyproject.toml#L181

Reply:
Yeah, I think we should mark some tests for dbt>=1.6.0 only.